### PR TITLE
i18n support

### PR DESCRIPTION
**Pull request by maggie-5miles**
Add i18n support to the project, and zh-CN locale resource as well.

### CODE REVIEW

**Comment by mikedemarais** [comment]
🦄🤠 i approve!

**Comment by haydenadams** [comment]
@maggie-5miles thank you maggie, this is fantastic! 

Had someone double check the Chinese - everything looks good. Will merge soon.

File: src/pages/Send/index.js [hunk 1]
```diff
@@ -641,31 +643,31 @@ class Send extends Component {
       return (
         <div>
           <div>
-            You are selling {b(`${+inputValue} ${inputLabel}`)}.
+            {t("youAreSending")} {b(`${+inputValue} ${inputLabel}`)}.
```
Comment (tpmccallum) [inline src/pages/Send/index.js]: Sending vs Selling
Please replace t("youAreSending") with t("youAreSelling")

Comment (tpmccallum) [inline src/pages/Send/index.js]: I originally thought this was supposed to be t("youAreSelling" i.e. selling vs sending)
However, I upon closer inspection I see that the account is actually "sending" (to the recipient account) so this is technically correct (for the Send) page.

Comment (maggie-5miles) [reply src/pages/Send/index.js]: Yes, I did that on purpose, hope it's right.

**Comment by mikedemarais** [comment]
@haydenadams 
i just ran this locally and can confirm the translations are displayed as intended.

if you would like to try it yourself, pull this branch locally, and just append `?lng=LANGUAGE` to the uniswap URL. this overrides the automatic language chosen by the i18next-browser-languageDetector package we are using.

ie: `localhost:3000/swap?lng=zh-CN`

**Pull request merged**